frontend/src/components/VCollectionHeader/VCollectionHeader.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <div class="flex flex-col gap-y-6 lg:gap-y-8">
+    <div class="flex flex-col gap-4 lg:flex-row lg:justify-between">
+      <div class="flex items-center gap-x-2">
+        <VIcon :name="iconName" :size="isDesktop ? 10 : 6" />
+        <h1 class="text-3xl font-semibold leading-6 lg:text-6xl lg:leading-10">
+          {{ title }}
+        </h1>
```

Comment:
In `md` I envision a white gap in the top-right area for most cases. To avoid that, I think we should try changing to `flex-row`  from `md` instead of `lg` .